msbuild/Xamarin.MacDev.Tasks/Tasks/CompileEntitlements.cs

```diff
@@ -224,19 +224,19 @@ PString MergeEntitlementString (PString pstr, MobileProvision? profile, bool exp
 			return null;
 		}
 
-		PDictionary MergeEntitlementDictionary (PDictionary dict, MobileProvision? profile)
+		PDictionary MergeEntitlementDictionary (PDictionary dict, MobileProvision? profile, string? key)
```

Comment:
Should we remove the `key` parameter as we always use the item's key instead?

Reply:
Thanks for the suggestion, I did as you suggested and removed the `key` parameter since it's not used.